src/project.cc

```diff
@@ -157,6 +156,8 @@ Project::Entry GetCompilationEntryFromCompileCommandEntry(
     return result;
 
   bool clang_cl = strstr(args[0].c_str(), "clang-cl") ||
+                  strstr(args[0].c_str(), "clang-cl.exe") ||
+                  strstr(args[0].c_str(), "cl") ||
```

Comment:
Do you mean `args[0] == "cl" || args[0] == "cl.exe"` instead of `strstr`?

Reply:
I just didn't read correctly what strstr actually does. I just added redundant checks, I'll remove them.